runtime/moonriver/src/lib.rs

```diff
@@ -1365,6 +1386,8 @@ construct_runtime! {
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 36,
 
 		// Sudo was previously index 40
+		#[cfg(feature = "force-debug")]
+		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 40,
```

Comment:
This seems to break the `construct_runtime!` macro when the feature is not set. So most likely the `#[cfg()]` is ignored by the macro.